CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
+<!-- changelog-linker -->
 
+## Unreleased
 # Changelog
 
```

Comment:
@samsonasik Hey, this headline position of autogenerated changelog seems wrong.
Could you look at it?

Reply:
I will try